consensus/state.go

```diff
@@ -1844,6 +1846,7 @@ func (cs *State) recordMetrics(height int64, block *types.Block) {
 	cs.metrics.NumTxs.Set(float64(len(block.Data.Txs)))
 	cs.metrics.TotalTxs.Add(float64(len(block.Data.Txs)))
 	cs.metrics.BlockSizeBytes.Set(float64(blockSize))
+	// cs.metrics.BlockTimeSeconds
```

Comment:
is this mean to be commented out still?

Reply:
Thank you for the review 👍 (the PR is till in draft mode though :) ). Yes, the part you mentioned was indeed intended to be removed. It is now fixed.